src/query/src/sql/show_create_table.rs

```diff
@@ -47,7 +49,12 @@ fn create_sql_options(table_meta: &TableMeta) -> OptionMap {
     }
     if let Some(ttl) = table_opts.ttl {
         options.insert(TTL_KEY.to_string(), format_duration(ttl).to_string());
-    }
+    } else if let Some(database_ttl) = schema_options.and_then(|o| o.ttl) {
```

Comment:
Kind of "gild the lily" to me here. I think if the table is created without the ttl options, then the "show create" is supposed to show its creation as is, no need to add any schema-level options to it.

Reply:
> Kind of "gild the lily" to me here. I think if the table is created without the ttl options, then the "show create" is supposed to show its creation as is, no need to add any schema-level options to it.

I discussed this with @fengjiachun and @WenyXu . If the database TTL actually takes effect, it should be reflected in SHOW CREATE TABLE statement.